Hippo.Web/Hippo.Web.csproj

```diff
@@ -14,7 +14,16 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="6.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.1">
+      <PrivateAssets>all</PrivateAssets>

```

Comment:
@srkirkland or @sprucely 
Know what this is?
Should I remove this section?

Reply:
Not certain, but I think that's to support some of Visual Studio's schema/query designer tools.

Reply:
I think you need it to be able to run the migration scripts, like dotnet ef add migration or whatever the syntax is